internal/handlers/pg/msg_getparameter.go

```diff
@@ -25,25 +26,80 @@ import (
 
 // MsgGetParameter OpMsg used to get parameter.
 func (h *Handler) MsgGetParameter(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error) {
-	document, err := msg.Document()
+	cmd, err := msg.Document()
```

Comment:
All handlers has `document, err := msg.Document()` as the first line. Let's keep it the same for consistency

Reply:
fix